ktor-server/ktor-server-plugins/ktor-server-status-pages/jvmAndNix/src/io/ktor/server/plugins/statuspages/StatusPages.kt

```diff
@@ -29,7 +29,14 @@ public val StatusPages: ApplicationPlugin<StatusPagesConfig> = createApplication
     val statuses = HashMap(pluginConfig.statuses)
 
     fun findHandlerByValue(cause: Throwable): HandlerFunction? {
-        val key = exceptions.keys.find { cause.instanceOf(it) } ?: return null
+        val keys = exceptions.keys.filter { cause.instanceOf(it) }
+        if (keys.isEmpty()) return null
+
+        if (keys.isNotEmpty() && keys.size == 1) {
```

Comment:
`keys.isNotEmpty()` is always false here

Reply:
thanks, will fix